graphrag/query/structured_search/global_search/search.py

```diff
@@ -277,6 +283,9 @@ async def _reduce_response(
 
             if len(filtered_key_points) == 0 and not self.allow_general_knowledge:
                 # return no data answer if no key points are found
+                log.warning(
+                    "Warning: All map responses have score 0, returning a canned 'I do not know' answer"
```

Comment:
Is it possible to return a suggestion on how to improve results alongside "I do not know"?

Reply:
I added a more elaborated message with suggestions.